### PR TITLE
sector visualization by activity & scope

### PR DESCRIPTION
**Pull request by thehighestprimenumber**
Show sector visualization breakdown

image
image
image
image

### CODE REVIEW

File: app/src/app/sectors.ts
Comment (lemilonkh) [inline app/src/app/sectors.ts:1]: Can you name this file or put it into a folder so it's clear that this is a utility file for the frontend defining the metadata for the different sectors?

**Review by lemilonkh — APPROVED** [review]
Thanks :sparkles:

File: app/src/app/[lng]/[inventory]/InventoryResultTab/ByScopeView.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  sectorName: string;
+}
+
+const ByScopeView: React.FC<ByScopeViewProps> = ({
+  data,
+  tData,
+  tDashboard,
+  sectorName,
+}) => {
+  const scopes = sectorName === "waste" ? ["1", "3"] : ["1", "2"];
```
[the remark this answers is not in the document]
Comment (thehighestprimenumber) [reply app/src/app/[lng]/[inventory]/InventoryResultTab/ByScopeView.tsx:29]: If I use the scopes from the db, I get
scopes 1,2,3 for transportation and stationary energy.
However, there's no way to add data to scope 3 in our current state of the system, so it feels weird to show a column that will always be empty. What should we do?

[the remark this answers is not in the document]
Comment (lemilonkh) [reply app/src/app/[lng]/[inventory]/InventoryResultTab/ByScopeView.tsx:29]: Fine for now then. Let's note it down and confirm with the design/ data teams which scopes should be shown.

**Pull request merged**